requirements/constraints.txt

```diff
@@ -20,4 +20,7 @@ twine<2.0.0
 zipp<2.0.0
 
 # Requires: Python >=3.6
-inflect<4.0.0
\ No newline at end of file
+inflect<4.0.0
+
+# Requires: Python >=3.8
```

Comment:
how did we confirm that this is the specific version needed? Is there a specific language feature used?